app/models/driver.rb

```diff
@@ -0,0 +1,5 @@
+class Driver < ApplicationRecord
+	validates :email, :uniqueness => { :case_sensitive => false }
```

Comment:
Teeny nitpick: 4 spaces rather than 2?

Reply:
@raghubetina, actually is this syntax still okay? Do we want the generator to produce more verbose code?

```ruby
  validates(:email, { :uniqueness => { :case_sensitive => false } })
```